src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,103 @@
+package duke.parser;
+
+import duke.command.CommandWord;
+import duke.exception.InvalidCommandFormatException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Todo;
+
+import static duke.constants.DukeCommandStrings.*;
+
+public class Parser {
+
+    public static CommandWord parseCommandWord(String userInput) {
+        if (beginsWith(userInput, LIST_COMMAND)) {
+            return CommandWord.LIST;
+        } else if (beginsWith(userInput, DONE_COMMAND)) {
+            return CommandWord.DONE;
+        } else if (beginsWith(userInput, TODO_COMMAND)) {
+            return CommandWord.TODO;
+        } else if (beginsWith(userInput, DEADLINE_COMMAND)) {
+            return CommandWord.DEADLINE;
+        } else if (beginsWith(userInput, EVENT_COMMAND)) {
+            return CommandWord.EVENT;
+        } else if (beginsWith(userInput, DELETE_COMMAND)) {
+            return CommandWord.DELETE;
+        } else if (beginsWith(userInput, HELP_COMMAND)) {
+            return CommandWord.HELP;
+        } else if (beginsWith(userInput, EXIT_COMMAND)) {
+            return CommandWord.EXIT;
+        } else {
+            return CommandWord.INVALID;
+        }
+    }
+
```

Comment:
to improve the readability of the code, consider converting this logic to switch case